mgmt_cli_test.py

```diff
@@ -710,7 +709,7 @@ def test_client_encryption(self):
                 DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR, line="get_repair_meta: repair_meta_id"):
 
             self.db_cluster.enable_client_encrypt()
-
+            repair_task = mgr_cluster.create_repair_task(fail_fast=True)
             repair_task.wait_for_status(list_status=[TaskStatus.ERROR, TaskStatus.ERROR_FINAL], step=5, timeout=240)
```

Comment:
task is waited to be error, cause it change from non encryption to encryption.

Reply:
Exactly, I am not expecting it to succeed, it is by design.